src/pages/CalendarPage/CalendarPage.tsx

```diff
@@ -47,6 +47,16 @@ function UpdateAvailabilityModal({
         setUserAvailabilityData(newUserAvailabilityData);
```

Comment:
Since we already have the zeroState defined for heatMapData, I think we should utilize that for the zeroState array

Reply:
I agree with you @ivancuadra869 but I think as it stands a little bit of refactoring would need to be done because I believe you would need to match the dimensions of the zero state to the actual size of the array.

Reply:
Refactored and committed this suggestion on top of this PR.